concepts/units-of-measurement.md

```diff
@@ -95,95 +97,135 @@ Imperial:
+| Dimensionless          | Decibel                          | dB     |
+| ElectricPotential      | Volt                             | V      |
+| ElectricCapacitance    | Farad                            | F      |
+| ElectricCharge         | Coulomb                          | C      |
+| ElectricConductance    | Siemens                          | S      |
+| ElectricCurrent        | Ampere                           | A      |
+| ElectricInductance     | Henry                            | H      |
+| ElectricResistance     | Ohm                              | Ω      |
+| Energy                 | Joule                            | J      |
+| Energy                 | Watt Second                      | Ws     |
+| Energy                 | Watt Hour                        | Wh     |
+| Energy                 | Kilowatt Hour                    | kWh    |
+| Energy                 | Megawatt Hour                    | MWh    |
+| Energy                 | Kilovar Hour                     | kvarh  |
+| Force                  | Newton                           | N      |
+| Frequency              | Hertz                            | Hz     |
+| Illuminance            | Lux                              | lx     |
+| Intensity              | Irradiance                       | W/m²   |
+| Intensity              | Microwatt per square Centimeter  | µW/cm² |
+| Length                 | Metre                            | m      |
+| Length                 | Kilometre                        | km     |
+| LuminousFlux           | Lumen                            | lm     |
+| LuminousIntensity      | Candela                          | cd     |
+| MagneticFlux           | Weber                            | Wb     |
+| MagneticFluxDensity    | Tesla                            | T      |
+| Mass                   | Kilogram                         | kg     |
+| Mass                   | Gram                             | g      |
+| Power                  | Watt                             | W      |
+| Power                  | Kilovar                          | kvar   |
+| Power                  | Decibel-Milliwatts               | dBm    |
+| Pressure               | Pascal                           | Pa     |
+| Pressure               | Hectopascal                      | hPa    |
+| Pressure               | Millimetre of Mercury            | mmHg   |
+| Pressure               | Bar                              | bar    |
+| Pressure               | Millibar                         | mbar   |
+| Radioactivity          | Becquerel                        | Bq     |
+| RadiationDoseAbsorbed  | Gray                             | Gy     |
+| RadiationDoseEffective | Sievert                          | Sv     |
+| SolidAngle             | Steradian                        | sr     |
+| Speed                  | Metre per Second                 | m/s    |
+| Speed                  | Kilometre per Hour               | km/h   |
```

Comment:
Can we add `mm/h` (see https://github.com/openhab/openhab-core/issues/1297)?

Reply:
Done.